app/res/xml/preferences_developer.xml

```diff
@@ -85,6 +85,11 @@
         android:entryValues="@array/pref_enabled_vals"
         android:key="cc-auto-purge"
         android:title="Detect and Trigger Purge on Form Save"/>
+    <EditTextPreference
```

Comment:
I agree but the main use case of this pref is to enable UI tests benchmark different app functionalities on a a bulk load of forms (Upload Sync for example). So dropping calabash support for this is not really an option.